iphone/Classes/KrollContext.h

```diff
@@ -175,3 +183,29 @@ KrollContext *GetKrollContext(TiContextRef context);
 @property (nonatomic, readwrite, retain) id invocationArg4;
 
 @end
+
+#if defined(USE_JSCORE_FRAMEWORK) && !defined(TI_USE_KROLL_THREAD)
+
+/**
+ * Handles creating and clearing timers when running with JavaScriptCore
+ */
+@interface JSTimerManager : NSObject
```

Comment:
Small peak: This may clash with internal API's some day, since it uses the JavaScriptCore framework prefix. To be save, naming it like `TiTimerManager` or `TiJSTimerManager` may help.